doc/lsst.daf.relation/index.rst

```diff
@@ -174,10 +174,10 @@ The standard approach to designs like this in object oriented programming is the
 This implicitly restricts the set of node tree types to those with methods in the algorithm/engine base class.
 
 In languages with functional programming aspects, a much more direct approach involving enumerated types and pattern-matching syntax is often possible.
-With the introduction of the `match` statement in Python 3.10, this now includes Python, and this is the approach taken in here.
-This results in much more readable and concise code than the boilerplate-heavy visitor-pattern alternative, but it comes with a significant drawback: there are no checks (either at runtime or via static type checkers like MyPy) that all necessary `case` branches of a `match` are present.
+With the introduction of the ``match`` statement in Python 3.10, this now includes Python, and this is the approach taken in here.
```

Comment:
Do you know if single backticks usually works for Python language keywords (as opposed to standard library symbols), and hence `match` and `case` might be missing just because they're new?  That's what I was originally assuming when I saw that these didn't render as links (i.e. I left them as single backticks because I thought they might just start working someday with a Sphinx upgrade), but I don't actually know if that's at all likely.

Reply:
It seems to be more fundamental for python statements:

```sh
$ pydoc match
No Python documentation found for 'match'.
```

so it's not surprising it won't link to anything (this is also true in python 3.11). If you are hopeful it might someday work I can add an ignore entry for it and  revert this change.

Reply:
Thanks, I didn't know about the `pydoc` trick.  And I see that `pydoc if` and `pydoc while` both yield useful results, so I think I am hopeful that these will someday get fixed.  If a config ignore entry isn't too hard, that'd be preferable, but if it's hard to define one that isn't too general your fix is fine, and your fix is probably better than a bunch of per-line ignores.

Reply:
It's not a per-line ignore. It would go in conf.py and be an ignore for any time someone references `match`. I'll do that.